### PR TITLE
Add --prefer-lowest to travis build matrix

### PR DESCRIPTION
**Pull request by kbond**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,4 +26,4 @@ install:
 
 script:
     - php bin/phpspec run
-    - php bin/phpunit
+    - php bin/simple-phpunit
```
Comment (umpirsky) [inline .travis.yml]: What is simple-phpunit? :)

Comment (kbond) [reply .travis.yml]: It is provided by symfony/phpunit-bridge. Allows you to run your test suite built with older versions of phpunit (in this case symfony 2.7)

**Comment by kbond** [comment]
For some reason `composer update --prefer-lowest` is hanging. Did this locally and looks like it is doing it on travis... not sure why.

**Comment by umpirsky** [comment]
@kbond I restarted the job, let's see...

Thanks!

**Comment by kbond** [comment]
Ok, I tried something else. symfony/framework-bundle 2.7.25 is the first version of symfony that allows phpunit v6 so I made that the min. Also, I allowed for older versions of phpspec due to a conflict with symfony 2.7 (this also makes the nightly job pass)

**Comment by kbond** [comment]
Looks like it is good now. As long as you don't mind the allowed min version of 2.7.25 on the framework-bundle. It is a little annoying that it is only to make the tests pass.

One option could be to allow add the 2.7.25 min requirement to `require-dev`...

**Comment by umpirsky** [comment]
@kbond I don't mind, I look forward, and there are older versions of the bundle if someone needs them.

Thanks!

**Comment by kbond** [comment]
Could you make a new tag?

**Comment by umpirsky** [comment]
Done. ;)

**Pull request merged**